app/controllers/categories_controller.rb

```diff
@@ -1,15 +1,35 @@
 class CategoriesController < ApplicationController
   before_action :set_category, only: [:show]
```

Comment:
This was how we handled this in other controllers so we wanted to do it this way for consistency, and to make it easy later on to add other actions.